### PR TITLE
docs: add Jules Kremer to Angular team. Removed old members.

### PR DESCRIPTION
**Pull request by GavinMK**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -293,15 +259,6 @@
...
   },
-  "manekinekko": {
-    "name": "Wassim Chegham",
-    "picture": "wassim.jpg",
-    "twitter": "manekinekko",
-    "website": "https://medium.com/@wassimchegham",
-    "bio": "Wassim (aka manekinekko on Twitter/Github) is a Developer Advocate at SFEIR, in Web technologies (Angular, Polymer, PWA, Web Components...). He is also a Developer Expert in Web technologies nominated by Google. He enjoys writing technical articles, meeting developers at events, speaking at conferences and contributing to open source projects. Wassim loves the Web Platform and works hard to move it forward.",
-    "groups": ["Angular"],
-    "mentor": "filipesilva"
-  },
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: @manekinekko, is also an [Angular GDE][1].
Could you update his entry to eflect that?

```diff
   "manekinekko": {
     ...
-    "groups": ["Angular"],
-    "mentor": "filipesilva"
+    "groups": ["GDE"]
   },
```

[1]: https://developers.google.com/community/experts/directory/profile/profile-wassim_chegham

**Review by gkalpak — CHANGES_REQUESTED** [review]
Awesome! Thx for the clean-up 💯
Could you also, please, remove the pictures (in [aio/content/images/bios][1]) that are no longer used? 🙏 
(It might be a good idea to split the changes into two separate commits: one cleaning up and one adding @juleskremer.)

##
(Unrelated to this PR: It would probably be a good idea for us to have an "Alumi" section 😃)

[1]: https://github.com/angular/angular/tree/ffe24e0fe38d51813ba7056668dc907604db76b0/aio/content/images/bios

**Comment by kapunahelewong** [comment]
Hi @GavinMK! Thanks so much for this. If you need a reference, I have [notes of my own workflow](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca) with steps for rebasing, amending, squashing, etc. Let me know when you're ready for the next step or if you have any questions.

File: aio/content/marketing/contributors.json [hunk 2]
```diff
@@ -821,5 +821,13 @@
     "bio": "Andrew is a software engineer at Google on the Angular Core team.",
     "groups": ["Angular"],
     "lead": "juleskremer"
+  },
+  "ManuMurthy": {
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: We should use GitHub handles as keys (see [aio/content/marketing/README.md][1]), so this should be [manughub][2].

[1]: https://github.com/angular/angular/blob/4fa3dc2b2/aio/content/marketing/README.md#about-the-data
[2]: https://github.com/manughub

File: aio/content/marketing/contributors.json [hunk 3]
```diff
@@ -814,5 +814,12 @@
     "twitter": "joeyperrott",
     "groups": ["Angular"],
     "lead": "igorminar"
+  },
+  "AndrewScott": {
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: We should use GitHub handles as keys (see [aio/content/marketing/README.md][1]), so this should be [atscott][2].

[1]: https://github.com/angular/angular/blob/4fa3dc2b2/aio/content/marketing/README.md#about-the-data
[2]: https://github.com/atscott

File: aio/content/marketing/contributors.json [hunk 4]
```diff
@@ -814,5 +814,12 @@
...
+    "name": "Andrew Scott",
+    "picture": "andrewscott.jpg",
+    "bio": "Andrew is a software engineer at Google on the Angular Core team.",
+    "groups": ["Angular"],
+    "lead": "juleskremer"
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Is this correct? (I would expect Kara or Igor.)

File: aio/content/marketing/contributors.json [hunk 5]
```diff
@@ -409,14 +374,6 @@
     "bio": "Filip is a Frontend developer from Denmark. He works at IMPACT, delivering large-scale, high-performance e-commerce to international clients - most often build in Angular. He runs the local Angular usergroup - ngAarhus, and gives talks/workshops around and beyond the country of Denmark.",
     "groups": ["GDE"]
   },
-  "ocombe": {
-    "name": "Olivier Combe",
-    "picture": "ocombe.jpg",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Can you please remove the image file as well?

File: aio/content/marketing/contributors.json [hunk 6]
```diff
@@ -566,14 +523,6 @@
     "bio": "Kim is an an Angular consultant, developer, speaker, writer, and Google Developer Expert. She is passionate about learning from and sharing knowledge with other developers through blogging, speaking, workshops, and open source.",
     "groups": ["GDE"]
   },
-  "elanaolson": {
-    "name": "Elana Olson",
-    "picture": "elanaolson.jpg",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Can you please remove the image file as well?

File: aio/content/marketing/contributors.json [hunk 7]
```diff
@@ -722,43 +671,19 @@
     "bio": "Justin (aka Schwarty) is a Google Developer Expert in Web Technologies and Angular, the host and maintainer of the weekly AngularAir live video broadcast, educator, writer and content creator. He has Angular courses available on LinkedIn Learning and Pluralsight and loves passing on years of full stack development knowledge to help empower others to find their inner awesomeness!",
     "groups": ["GDE"]
   },
-  "brandonroberts": {
-    "name": "Brandon Roberts",
-    "picture": "brandonroberts.jpg",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Can you please remove the image file as well?

File: aio/content/marketing/contributors.json [hunk 8]
```diff
@@ -722,43 +671,19 @@
...
-    "twitter": "brandontroberts",
-    "website": "https://brandonroberts.dev",
-    "bio": "Brandon is a developer and technical writer working on guides, tutorials, application development, and infrastructure for the Angular docs. He is also a maintainer of the NgRx project, building reactive libraries for Angular.",
-    "groups": ["Angular"],
-    "lead": "dennispbrown"
-  },
-  "chembu": {
-    "name": "Sreevani Sreejith",
-    "picture": "sreevani.jpg",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Can you please remove the image file as well?

File: aio/content/marketing/contributors.json [hunk 9]
```diff
@@ -722,43 +671,19 @@
...
   },
   "jbogarthyde": {
     "name": "Judy Bogart",
     "picture": "judy.png",
     "groups": ["Angular"],
     "lead": "dennispbrown"
   },
-  "jenniferfell": {
-    "name": "Jennifer Fell",
-    "picture": "jennifer.jpg",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Can you please remove the image file as well?

**Review by gkalpak — CHANGES_REQUESTED** [review]
This looks great 🎉
Just a little clean-up and it should be good to go.

<sub>You don't have to, but -if you fancy an extra challenge- you could split the removal of old members (incl. their pictures) in one commit and the additions/updates in another (or even each addition/update in its own commit) :wink:</sub>

**Comment by gkalpak** [comment]
😢

**Pull request closed without merging**